src/platform/backtrace.cpp

```diff
@@ -66,31 +90,48 @@ void bt_impl( std::ostream& strm )
 }
 #endif
 
-void saveBacktrace()
+boost::filesystem::path saveBacktrace()
 {
     // kind of ugly, prevent additional bts generated after incorrect termination ( registered_id asserts, etc )
     static bool handled = false;
     if( handled )
     {
-        return;
+        return boost::filesystem::path();
     }
     handled = true;
     std::string const& tempmodel = boost::filesystem::temp_directory_path().string();
     boost::filesystem::path temp = boost::filesystem::unique_path( tempmodel + "/reaping2-\%\%\%\%-\%\%\%\%-\%\%\%\%-\%\%\%\%.txt" );
     std::string tempstr = temp.string();
     std::ofstream ofs( tempstr.c_str() );
+    ofs << GIT_VERSION << "\n";
+    ofs << GIT_DATE << "\n";
+    ofs << GIT_BRANCH << "\n";
+    ofs << GIT_REMOTE << "\n";
     bt_impl( ofs );
+    return temp;
 }
 
 void handle( int signal )
 {
-    saveBacktrace();
+    boost::filesystem::path const& fname = saveBacktrace();
+    if( !fname.empty() )
+    {
+        std::vector<std::string> args;
+        args.push_back( fname.string() );
+        platform::process::Start( ( exeDir() / exeName ).string(), args );
+    }
     abort();
 }
 
 void handle_exc()
 {
-    saveBacktrace();
+    boost::filesystem::path const& fname = saveBacktrace();
+    if( !fname.empty() )
+    {
+        std::vector<std::string> args;
+        args.push_back( fname.string() );
+        platform::process::Start( ( exeDir() / exeName ).string(), args );
+    }
     abort();
 }
```

Comment:
Why are there two functions with the exact same implementation? Why not call one from the other?